ParisBNF/abb/BNFabb97.xml

```diff
@@ -0,0 +1,1357 @@
+                <locus from="96rb" to="96va"/>
+                <title>ሥርዓተ፡ ዘቤተ፡ ንጉሥ፡ </title>
+                <textLang mainLang="gez"/>
+                <incipit><hi rend="rubric">ሥርዓተ፡ ዘቤተ፡ ንጉሥ፡ ዘይቀድሙ፡ </hi>ድሙ፡ ለምክር፡ ወቅድመ፡ ኵሉ፡ ዘይመክር፡
+                  የሻለቃ፡ ዘይል፡ ጭፍራ፡ ወበላዕሌሁ፡ ይመክር፡ የሽአለቃ፡ ዘተሠርዓ፡ ጭፍራ፡ በሀገር፡ በበመዓርጊሁ፡ ወበበጾታሁ።</incipit>
+                <explicit>ወበላዕሌሆሙ፡ ይመክር፡ ንጉሥ፡ ኍሉቀ፡ ወምቱረ፡ ነገረ። ። </explicit>
+              </msItem>
+              <msItem xml:id="ms_i24.2">
+                <locus from="96va" to="99rb"/>
+                <title>ሥርዓት፡ ወሕግ፡ ዘቤተ፡ ንጉሥ፡ አመ፡ ሞተ፡ ንጉሥ፡</title>
+                <incipit><hi rend="rubric">ሥርዓት፡ ወሕግ፡</hi> ዘቤተ፡ ንጉሥ፡ አመ፡ ሞተ፡ ንጉሥ፡ ያንብሩ፡ አስከሬነ፡ ኀበ፡
+                  ዐቢይ፡ ቤቱ። </incipit>
+                <explicit>ዘወሰኩ፡ ለፍትሕ፡ በመንግሥተ፡ ዓምደ፡ ጽዮን። ቀዳሚሰ፡ ነበሩ፡ እንዘ፡ ይነዝኁ፡ ማየ፡ ጸሎት፡ ጽራጅ፡ ማሰሬ፡
+                  ዘያነብር፡ አክሊለ፡ ዲበ፡ ርእሰ፡ ንጉሥ። ለዛቲ፡ መጽሐፍ፡ አጽሐፍኖ፡ ቀደምት፡ በእንተ፡ ፍቅረ፡ ዚአሃ፡ ለጽዮን፡ ሰማያዊት፡
+                  ዘሀለወት፡ አክሱም፡</explicit>
+              </msItem>
+            </msItem>
+          </msContents>
+          <physDesc>
+            <objectDesc form="Codex">
+              <supportDesc>
+                <support>
+                  <material key="parchment"/>
+                </support>
+                <extent>
+                  <measure unit="leaf">99</measure>
+                  <dimensions type="outer">
+                    <height unit="mm">280</height>
+                    <width unit="mm">220</width>
+                  </dimensions>
+                </extent>
+                <foliation>Foliation in the upper right corner of the folia.</foliation>
+                <collation>
+                  <list>
+                    <item xml:id="q1" n="1">
+                      <num value="1">፩</num>
+                      <dim unit="leaf">10</dim>
+                      <locus from="1r" to="11v"/>
+                    </item>
+                    <item xml:id="q2" n="2">
+                      <num value="2">፪</num>
+                      <dim unit="leaf">10</dim>
+                      <locus from="12r" to="21v"/>
+                    </item>
+                    <item xml:id="q3" n="3">
+                      <num value="3">፫</num>
+                      <dim unit="leaf">10</dim>
+                      <locus from="22r" to="31v"/>
+                    </item>
+                    <item xml:id="q4" n="4">
+                      <num value="4">፬</num>
+                      <dim unit="leaf">8</dim>
+                      <locus from="32r" to="39v"/>
+                    </item>
+                    <item xml:id="q5" n="5">
+                      <num value="5">፭</num>
+                      <dim unit="leaf">10</dim>
+                      <locus from="40r" to="49v"/>
+                    </item>
+                    <item xml:id="q6" n="6">
+                      <num value="6">፮</num>
+                      <dim unit="leaf">7</dim>
+                      <locus from="50r" to="57v"/>
```

Comment:
Can you establish the structure of the quires (stubs?)? The current encoding will lead to an error, because there is no explanation for the uneven number of leaves. But if this is not visible from the images, it can stay like this.

Reply:
No, I can´t! I have established the quires based on the presence of quire marks. Should I state it more clearly?